docs/OntologiesReviewWorkflow

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc
+title: Ontology Review Process
+---
+
+This page deals with the process, policies, and guidelines for manually reviewing newly-submitted ontologies, with particular focus on operational aspects such as:
+
+- which ontologies are reviewed, and in what order?
+- how are reviewers chosen?
+- what is the workflow for conducting a review?
+- what are the criteria used?
+
+Ontologies are reviewed manually after completing the automated evaluation. Though the automated evaluation checks for adherence to [OBO Foundry principles](http://www.obofoundry.org/principles/fp-000-summary.html), it cannot capture certain nuances outlined in the principles nor evaluate ontology quality. The purpose of the manual review is to check the ontology for issues that the NOR Dashboard does not cover.
+
+# Steps prior to manual review
+
+Once an ontology is submitted to the OBO Foundry, it is added to the New Ontology Dashboard for automated evaluation. The steps involved in ontology submission and subsequent dashboard evaluation are described in detail in this [FAQ answer](http://obofoundry.org/faq/how-do-i-register-my-ontology.html) and links therein.
```

Comment:
> it is added to the New Ontology Dashboard

As a reader, I would be curious how this happens -- is it completely automatic, or does a human have to add it?

Reply:
I suspect it is added by a person, but comments to this effect are outside the scope of this document. I only added a very brief synopsis of 'prior' steps because the dashboard is referred to later in the document, and I don't like bringing things up without prior introduction.